src/System.Diagnostics.DiagnosticSource/tests/DiagnosticSourceTests.cs

```diff
@@ -372,6 +372,12 @@ public void AllListenersAddRemove()
                 DiagnosticListener returnedListener = null;
                 Action<DiagnosticListener> onNewListener = delegate (DiagnosticListener listen)
                 {
+                    // Other tests can be running concurrently with this test, which will make
```

Comment:
If you want to disable that, you can, e.g. https://github.com/dotnet/corefx/blob/master/src/System.Linq.Parallel/tests/XunitAssemblyAttributes.cs#L10